rest/src/main/kotlin/builder/component/SelectMenuBuilder.kt

```diff
@@ -16,9 +17,10 @@ import kotlin.contracts.contract
  * @param customId The identifier for the menu, max 100 characters.
  */
 @KordDsl
-public class SelectMenuBuilder(
+public open class SelectMenuBuilder(
     public var customId: String,
 ) : ActionRowComponentBuilder() {
```

Comment:
same thing as with `SelectMenuComponent` in core: deprecate the public constructor and replace with sealed class in the future